guide/english/php/sessions/index.md

```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
 
 #### More Information:
 * <a href="https://secure.php.net/manual/en/book.session.php">php.net session manual</a>
+
```

Comment:
As a suggestion, firstly, when you are referencing codeignintqr here there is a need to give a basic definition of the same, not all php  beginners will know of the codeigniter framework leading them to do a google search removing their focus away from the article.

Add a small description about codeigniter first.